src/material/tabs/tabs.md

```diff
@@ -85,22 +85,36 @@ duration can be configured globally using the `MAT_TABS_CONFIG` injection token.
                "region": "slow-animation-duration"}) -->
 
 ### Accessibility
-`<mat-tab-group>` and `<mat-nav-tab-bar>` use different interaction patterns. The
-`<mat-tab-group>` component combines `tablist`, `tab`, and `tabpanel` into a single component with
-the appropriate keyboard shortcuts. The `<mat-nav-tab-bar>`, however, use a _navigation_ interaction
-pattern by using a `<nav>` element with anchor elements as the "tabs". The difference
-between these two patterns comes from the fact one updates the page URL while the other does not.
+`MatTabGroup` and `MatTabNavBar` imlement different interaction patterns for different use-cases.
```

Comment:
imlement -> implement

Reply:
The `<mat-nav-tab-bar>`, however, use a _navigation_ interaction

Should be "uses a_navigation_interaction"